reference/lib/ReferenceOrderCombiner.sol

```diff
@@ -238,9 +238,11 @@ contract ReferenceOrderCombiner is
             // Otherwise, track the order hash in question.
             orderHashes[i] = orderHash;
 
-            // Decrement the number of fulfilled orders.
-            maximumFulfilled--;
-
+            unchecked {
```

Comment:
Tiny nit: move the comment as to why unchecked is safe to right before the unchecked block and revise to `// Skip overflow check as maximumFulfilled is nonzero.`

Reply:
Done.